reactivesocket-cpp/src/Frame.h

```diff
@@ -121,14 +125,16 @@ class FrameMetadata {
   explicit FrameMetadata()
   : metadataPayload_(folly::make_unique<folly::IOBuf>()) {}
 
-  std::unique_ptr<folly::IOBuf> serialize();
+  void serializeInto(folly::io::QueueAppender& app);
```

Comment:
note that we cannot use Writable<T> here because we need a method only defined on QueueAppender.